pythonsdk/src/main/python/vantiqservicesdk.py

```diff
@@ -148,14 +149,33 @@ def __complete_request(completed_task):
                         await websocket.send_bytes('pong'.encode("utf-8"))
                         continue
 
-                    # Spawn a task to process the message and send the response
-                    task = asyncio.create_task(self.__process_message(websocket, msg_bytes))
-
-                    # Add the task to the set of active requests and remove when done.
-                    # See https://docs.python.org/3/library/asyncio-task.html#creating-tasks
-                    active_requests.add(task)
-                    self._active_requests.inc(1)
-                    task.add_done_callback(__complete_request)
+                    # Decode the message as JSON, but make sure to catch any errors
+                    try:
+                        request = json.loads(msg_bytes.decode("utf-8"))
+                    except Exception as e:
+                        # couldn't deserialize the request, send back an error, but there's no way to know the requestId
```

Comment:
when deser fails, we don't get any requestId to use with the error response. not sure how useful the error response is in this situation